lib/main.dart

```diff
@@ -265,102 +35,15 @@ Future<void> main() async {
     },
   );
 
-  final mediaConstraints = {
-    'audio': true,
-    'video': {
-      'mandatory': {
-        'minWidth': '640', // Provide your own width, height and frame rate here
-        'minHeight': '480',
-        'minFrameRate': '30',
-      },
-      'facingMode': 'user',
-      'optional': [],
-    }
-  };
-  final localStream = await navigator.getUserMedia(mediaConstraints);
+  final localStream = await useProvider(localStreamProvider.future);
```

Comment:
this almost certainly doesn't want to live here, but I'm too tired to move it right now